strapi/strapi
# feat(blocks): add keyboard shortcuts for modifiers

### What does it do?

Added keyboard shortcuts for modifiers

### How to test it?

User should be able to use common shortcuts like:

- Ctrl(Cmd) + b: Transform the selection into bold. 

- Ctrl(Cmd) + i: Transform the selection into italic. 

- Ctrl(Cmd) + u: Transform the selection into underline. 

- Ctrl(Cmd) + v: Add a link to the selection.

- Ctrl(Cmd) + e: Transform the selection into inline code.

- Ctrl(Cmd) + Shift + s: Transform the selection into strikethrough.
